client/pages/artist/_itemId/index.vue

```diff
@@ -120,8 +127,10 @@ import imageHelper from '~/mixins/imageHelper';
 import timeUtils from '~/mixins/timeUtils';
 import itemHelper from '~/mixins/itemHelper';
 import { isValidMD5 } from '~/utils/items';
+import BlurhashImage from '~/components/Layout/Images/BlurhashImage.vue';
```

Comment:
This is unnecessary

Reply:
Automatic import by VSCode. Removed in 576fdf1